kit/Kit.cpp

```diff
@@ -821,9 +826,12 @@ class Document final : public DocumentManagerInterface
     static void GlobalCallback(const int type, const char* p, void* data)
     {
         if (SigUtil::getTerminationFlag())
-        {
             return;
-        }
```

Comment:
Just nit-picking, but why such cosmetic changes in a functional PR?

Reply:
to make it more consistent =) [and sensible]